node/core/approval-voting/src/tests.rs

```diff
@@ -394,7 +448,7 @@ fn test_harness<T: Future<Output = VirtualOverseer>>(
 		ApprovalVotingSubsystem::with_config(
 			Config{
 				col_data: test_constants::TEST_CONFIG.col_data,
-				 slot_duration_millis: 100u64,
+				slot_duration_millis: 5000u64,
```

Comment:
Why did this need to change? Time should be totally programmatic in the tests. Also recommend extracting to a constant `TEST_SLOT_DURATION` or something.